### DISCUSSION TITLE
Add Sections to Ingredient List in a Recipe

### DISCUSSION THREAD

**Discussion by clappdj** [mealie-recipes/mealie#4114] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I'd like to be able to specify sections to help organize ingredients for certain recipes. For example, a lot of baking recipes have several components that come together into a single "recipe". A similar concept to this screenshot is what I'm after: 
![image](https://github.com/user-attachments/assets/64e81381-ce23-4433-87e9-c384b6bea45e)

### Please provide a concise description of the feature that would resolve your issue.

Users are able to organize an ingredient list into sections for each component of a recipe.

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This is already a feature:

![image](https://github.com/user-attachments/assets/29647cf0-6aa9-4253-be90-0e6b88e55044)
![image](https://github.com/user-attachments/assets/49869426-ba1f-46c6-92d5-4ad65f0b463c)
